README.md

```diff
@@ -44,6 +44,20 @@ docker-compose up
 
 This builds an image for your app and starts it, along with a linked container for MongoDB. Go to [http://localhost/](http://localhost/) to see your app running.
 
+### Troubleshooting
+
+When this repository is updated for new Meteor versions, all the images for versions >= 1.9 are rebuilt and republished.  So all existing images will get the latest ubuntu base image and other dependencies, so that you’re not using some years-old version of Ubuntu etc. with known security issues.
+Consequently if you have existing environments you may need to update your Ubuntu base image to ensure compatibility with this package.  To update your local docker:
```

Comment:
I don’t understand why this is necessary. In `example/default.dockerfile` and the other example, there’s no reference to Ubuntu; so why does it matter what versions of the Ubuntu Docker image are present in a user’s local Docker registry? How does pulling the latest help?

Reply:
I thought it was using that as a machine image base but could be wrong, the root of my problem was AWS machine base Ubuntu image so I thought it would also apply to local which someone else showed a screenshot of.  Anyways my problem is solved by the above advice, feel free to edit it if it doesn't make sense.

Reply:
I honestly don't understand how it helps. I'm not trying to criticize, I'm just trying to get to the bottom of it. I don't want the readme to raise more questions than it answers.

Are you sure it was only the Ubuntu base image that you updated? What about the Node base image that is referenced in the second stage of your app's Dockerfile?